src/app_manager/app_manager.py

```diff
@@ -459,7 +459,8 @@ def handle_start_app(self, req):
                     if 'run' in plugin and plugin['run']:
                         p, a = roslib.names.package_resource_name(plugin['run'])
                         args = plugin.get('run_args', None)
-                        node = roslaunch.core.Node(p, a, args=args, output='screen')
+                        node = roslaunch.core.Node(p, a, args=args, output='screen',
+                                                   required=False)
```

Comment:
do we need this?

Reply:
No, the `required` argument already defaults to False. This is added for increased clarity to the below check_required definition.